OpenSim/Simulation/Model/PhysicalFrame.h

```diff
@@ -151,19 +151,32 @@ class OSIMSIMULATION_API PhysicalFrame : public Frame {
     * the WrapObject.
     */
     void addWrapObject(WrapObject* wrapObject);
-    ///@} 
+    ///@}
 
-protected:
     /** @name Advanced: PhysicalFrame Developer Interface
     These methods are intended for PhysicalFrame builders. */
     ///@{
     /**
     Specify the MobilizedBody that implements this PhysicalFrame in the
     underlying MultibodySystem. The PhysicalFrame's MobilizedBodyIndex must be
     set by the end of PhysicalFrame::addToSystem()
-        */
+    */
     void setMobilizedBodyIndex(const SimTK::MobilizedBodyIndex& mbix) const;
```

Comment:
This was made `public`, so that various `PhysicalFrame` implementations can call it on eachover.

It's up for debate whether we want to tackle this differently (however, no approach should ever mention a specific frame implementation, so `friend class PhysicalFrame` is a no no)

Reply:
In the latest version of this PR, this method is no longer public (`setMobilizedBodyIndexOf` was added, instead).

`friend`ing wouldn't work in this case. `friend`ing a base class doesn't give derived classes the same association. It would only work if you had `friend StationDefinedFrame`, `friend PhysicalOffsetFrame`, etc., which would make the design brittle when adding new frame implementations (the proposed implementation means that new frame definitions can come along and have everything they need)